products/compute/api.yaml

```diff
@@ -3329,6 +3329,12 @@ objects:
         send_empty_value: true
         update_verb: :PATCH
         update_url:  projects/{{project}}/regions/{{region}}/forwardingRules/{{name}}
+      - !ruby/object:Api::Type::Boolean
+        name: 'isMirroringCollector'
+        min_version: beta
+        description: |
+          Indicates whether or not this load balancer can be used as a collector for packet mirroring.
```

Comment:
We should probably include the rest of the description on this about it only being applicable for INTERNAL rules.

From the docs: 
`This can only be set to true for load balancers that have their loadBalancingScheme set to INTERNAL.`

Reply:
Good point, have added that line. Thanks